docs/fidesops/docs/guides/reporting.md

```diff
@@ -77,60 +78,64 @@ Check out the [API docs here](/fidesops/api#operations-Privacy_Requests-get_requ
 
 ## View Individual Privacy Request Log Details
 
-Use the `verbose` query param to see more details about individual queries run as part of the Privacy Request along
-with individual statuses. 
+Use the `verbose` query param to see more details about individual collections visited as part of the Privacy Request along
+with individual statuses. Individual collection statuses include `in_processing`, `retrying`, `complete` or `error`.
+You may see multiple logs for each collection as they reach different steps in the lifecycle.  
 
 `verbose` will embed a “results” key in the response, with execution logs grouped by dataset name.  In the example below,
-we have two datasets: `my-mongo-db` and `my-postgres-db`. There is one execution log for my-mongo-db and two execution
-logs for my-postgres-db.  The embedded execution logs are automatically truncated at 50 logs, so to view the entire 
-list of logs, visit the execution logs endpoint separately.
+we have two datasets: `my-mongo-db` and `my-postgres-db`. There are two execution logs for my-mongo-db (when the `flights` 
```

Comment:
Should have `s around the second `my-mongo-db`

Reply:
Fixed!